.travis.yml

```diff
@@ -1,11 +1,10 @@
 sudo: false
 language: node_js
-cache:
-  directories:
-    - node_modules
+cache: yarn
```

Comment:
Can you explain this change?

Reply:
Well, yarn is not used on CI but npm, see https://travis-ci.org/final-form/react-final-form/builds/384784946#L450